src/index.html

```diff
@@ -1,13 +1,59 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="container container--card" data-qa="card">
+      <img
+        class="container__picture"
+        src="images/imac.jpeg"
+        alt="picture"
+      >
+      <a
+        href="/"
+        class="container__link-1"
+        data-qa="hover"
+      >
+        A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </a>  
+      <p class="container__productCode">Product code: 195434</p>
+
+      <div class="starss">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <span class="starss__span">Reviews: 5</span>
+      </div>
+        
+      <div class="container__price">
+        <span class="container__priceName">Price:</span>
+        <span class="container__priceNumber">$2,199</span>
+      </div>
+
+      <a
+        href="/"
+        class="container__link-2"
+        data-qa="hover"
+      >
+      
+        BUY
```

Comment:
```suggestion
      <a
        href="/"
        class="container__link-2"
        data-qa="hover"
      >
        BUY
```
and do not name classes with number
it will be not clear for code reviewer what is the difference between 1 and 2